sqlx-sqlite/src/connection/worker.rs

```diff
@@ -8,18 +8,22 @@ use futures_intrusive::sync::{Mutex, MutexGuard};
 use sqlx_core::sql_str::SqlStr;
 use tracing::span::Span;
 
+#[cfg(feature = "offline")]
 use sqlx_core::describe::Describe;
 use sqlx_core::error::Error;
 use sqlx_core::transaction::{
     begin_ansi_transaction_sql, commit_ansi_transaction_sql, rollback_ansi_transaction_sql,
 };
 use sqlx_core::Either;
 
+#[cfg(feature = "offline")]
 use crate::connection::describe::describe;
 use crate::connection::establish::EstablishParams;
 use crate::connection::execute;
 use crate::connection::ConnectionState;
-use crate::{Sqlite, SqliteArguments, SqliteQueryResult, SqliteRow, SqliteStatement};
+#[cfg(feature = "offline")]
+use crate::Sqlite;
```

Comment:
Instead of adding `#[cfg]` to imports, which tends to confuse IDEs, I prefer to just use fully qualified names where necessary.